renovate.json

```diff
@@ -4,7 +4,8 @@
     "openshift/s3-backup/docker/poetry.lock"
   ],
   "ignoreDeps": [
-    "axios"
+    "axios",
```

Comment:
```suggestion
```
Nice, I think we can remove axios now, it was updated past the breaking change by @jmetev1

Reply:
https://github.com/bcgov/wps/pull/3752/commits/3128177cd6eac47a6dde15e91918febf1f439d94